hs_core/tests/api/rest/test_resource_science_meta.py

```diff
@@ -38,29 +38,14 @@ def test_put_scimeta(self):
             ],
             "contributors": [{
```

Comment:
May be my comment was confusing. I was suggesting that we should test one creator/contributor by specifying data for all the attributes and then create another creator/contributor by specifying data for only the required fields (name or organization). If you think that kind of testing is not needed here I am fine with that too.